node/lib/task.ts

```diff
@@ -164,6 +192,7 @@ export function setResourcePath(path: string): void {
         resourceFile = null;
         libResourceFileLoaded = false;
         locStringCache = {};
+        resourceCulture = 'en-US';
```

Comment:
the resourceCulture is global variable, since the all L0 tests runs at the same process, i need cleanup those global vars.